shoes-swt/lib/shoes/swt/text_block/text_font_factory.rb

```diff
@@ -44,7 +44,9 @@ def build_font(name, size, styles)
       end
 
       def styles_bitmask(styles)
-        styles.reduce { |acc, elem| acc | elem }
+        styles.reduce do |acc, elem|
+          acc | elem
+        end
```

Comment:
:wave: 

Sorry what rubocop offence it this? To me one line blocks with `{` look quite fine?

Reply:
Wondered the same thing @spmarisa. If you could fill us in we can move this forward. Otherwise, I'll close it out when the next release comes around.